hazelcast-sql/pom.xml

```diff
@@ -73,6 +73,20 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
```

Comment:
What is the reason of this change? It is not needed for compilation in IDE. Is it required by Jenkins?